src/pages/dataElements/fields/AggregationTypeField.tsx

```diff
@@ -51,19 +53,36 @@ export function AggregationTypeField() {
 
     return (
         <FieldRFF
+            // See https://final-form.org/docs/react-final-form/types/FieldProps#validate
+            // for why this is necessary
+            key={disabled ? 0 : 1}
             disabled={disabled}
             component={SingleSelectFieldFF}
             dataTest="dataelementsformfields-aggregationtype"
-            required
+            required={!disabled}
             inputWidth="400px"
             name="aggregationType"
-            label={i18n.t('{{fieldLabel}} (required)', {
-                fieldLabel: i18n.t('Aggregation type'),
-            })}
+            label={
+                disabled
+                    ? i18n.t('Aggregation type')
+                    : i18n.t('{{fieldLabel}} (required)', {
+                          fieldLabel: i18n.t('Aggregation type'),
+                      })
+            }
             helpText={helpText}
             options={options || []}
             validateFields={[]}
-            validate={required}
+            // @TODO: Why can I not use `FormValues` here?
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            validate={(value: string, values: any) => {
```

Comment:
```suggestion
            validate={(value?: string, values: any) => {
```
Should fix the `Component` type-error

Reply:
Ah, yes of course.. Adding ` | undefined` just works!

---

Where does the `AllValues` type come from? It's not exported by `final-form` or `react-final-form`.

Reply:
Sorry I meant `FormValues`